csharp/tests/Valkey.Glide.IntegrationTests/SortedSetCommandTests.cs

```diff
@@ -200,4 +200,176 @@ public async Task TestSortedSetAdd_ObsoleteOverloads(BaseClient client)
     }
+
+    [Theory(DisableDiscoveryEnumeration = true)]
+    [MemberData(nameof(Config.TestClients), MemberType = typeof(TestConfiguration))]
+    public async Task TestSortedSetRemove_SingleMember(BaseClient client)
+    {
+        string key = Guid.NewGuid().ToString();
+
+        // Test removing from non-existent key
+        Assert.False(await client.SortedSetRemoveAsync(key, "member1"));
+
+        // Add members first
+        Assert.True(await client.SortedSetAddAsync(key, "member1", 10.5));
+        Assert.True(await client.SortedSetAddAsync(key, "member2", 8.2));
+
+        // Test removing existing member
+        Assert.True(await client.SortedSetRemoveAsync(key, "member1"));
+
+        // Test removing already removed member
+        Assert.False(await client.SortedSetRemoveAsync(key, "member1"));
+
+        // Test removing non-existent member
+        Assert.False(await client.SortedSetRemoveAsync(key, "nonexistent"));
+
+        // Verify remaining member still exists
+        Assert.True(await client.SortedSetRemoveAsync(key, "member2"));
+    }
+
+    [Theory(DisableDiscoveryEnumeration = true)]
+    [MemberData(nameof(Config.TestClients), MemberType = typeof(TestConfiguration))]
+    public async Task TestSortedSetRemove_MultipleMembers(BaseClient client)
+    {
+        string key = Guid.NewGuid().ToString();
+
+        // Test removing from non-existent key
+        Assert.Equal(0, await client.SortedSetRemoveAsync(key, ["member1", "member2"]));
+
+        // Add members first
+        var entries = new SortedSetEntry[]
+        {
+            new("member1", 10.5),
+            new("member2", 8.2),
+            new("member3", 15.0),
+            new("member4", 12.0)
+        };
+        Assert.Equal(4, await client.SortedSetAddAsync(key, entries));
+
+        // Test removing multiple existing members
+        Assert.Equal(2, await client.SortedSetRemoveAsync(key, ["member1", "member3"]));
+
+        // Test removing mix of existing and non-existing members
+        Assert.Equal(1, await client.SortedSetRemoveAsync(key, ["member2", "nonexistent", "member5"]));
+
+        // Test removing already removed members
+        Assert.Equal(0, await client.SortedSetRemoveAsync(key, ["member1", "member2"]));
+
+        // Verify only member4 remains
+        Assert.Equal(1, await client.SortedSetRemoveAsync(key, ["member4"]));
+    }
+
+    [Theory(DisableDiscoveryEnumeration = true)]
+    [MemberData(nameof(Config.TestClients), MemberType = typeof(TestConfiguration))]
+    public async Task TestSortedSetRemove_EmptyArray(BaseClient client)
+    {
+        string key = Guid.NewGuid().ToString();
+
+        // Add some members first
+        Assert.True(await client.SortedSetAddAsync(key, "member1", 10.5));
+
+        // Test removing empty array
+        var emptyMembers = Array.Empty<ValkeyValue>();
+        Assert.Equal(0, await client.SortedSetRemoveAsync(key, emptyMembers));
```

Comment:
Are we able to just do
```suggestion
        Assert.Equal(0, await client.SortedSetRemoveAsync(key, []));
```
or does that cause a compiler and/or linter error

Reply:
Updated.